test/python/providers/test_fake_backends.py

```diff
@@ -103,6 +103,12 @@ def test_to_dict_properties(self, backend):
         else:
             self.assertTrue(backend.configuration().simulator)
 
+    @data(*FAKE_PROVIDER_FOR_BACKEND_V2.backends())
+    def test_convert_to_target(self, backend):
+        target = backend.target
+        if target.dt is not None:
+            self.assertLess(target.dt, 1e-6)
```

Comment:
This is fine for now but if any of the fake backends that start using a different dt this will cause a failure when we add them. It might be worth adding a comment here but not critical.

Reply:
Happy to do that if you think it'd be helpful. This is copied from the test below it for V1 providers.

Reply:
I'll just leave it as is since I think it'll be pretty clear if there's a failure here for a new backend :).